src/main/resources/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfig/config.jelly

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:section title="${%Amazon S3 Bucket Access settings}">
+        <f:entry title="${%S3 Bucket Name}" field="container">
+            <f:textbox/>
+        </f:entry>
+        <f:entry title="${%S3 Bucket Region}" field="region">
+            <f:select/>
+        </f:entry>
+        <f:entry title="${%Base folder}" field="prefix">
+            <f:textbox/>
+        </f:entry>
+        <f:entry title="${%Delete Blods}" help="help-deleteblods.html">
```

Comment:
Two typos. Anyway the `help` attribute can be elided. Simply name your help file properly: `help-deleteBlobs.html`. And use the `field` attribute.